src/sentry/integrations/msteams/webhook.py

```diff
@@ -200,6 +202,55 @@ def handle_member_removed(self, request):
         integration.delete()
         return self.respond(status=204)
 
+    def make_action_data(self, identity, data):
+        action_data = {}
+        if data["actionType"] == "unresolve" or data["actionType"] == "unignore":
+            action_data = {"status": "unresolved"}
+            analytics.record("integrations.msteams.unresolve", actor_id=identity.user_id)
+        elif data["actionType"] == "resolve":
+            status = data["resolveInput"]
+            status_data = status.split(":", 1)
+            resolve_type = status_data[-1]
+
+            action_data = {"status": "resolved"}
+            if resolve_type == "inNextRelease":
+                action_data.update({"statusDetails": {"inNextRelease": True}})
+            elif resolve_type == "inCurrentRelease":
+                action_data.update({"statusDetails": {"inRelease": "latest"}})
+            analytics.record(
+                "integrations.msteams.resolve", actor_id=identity.user_id, resolve_type=resolve_type
+            )
+        elif data["actionType"] == "ignore":
+            action_data = {"status": "ignored"}
+            ignore_count = int(data["ignoreInput"])
+            if ignore_count > 0:
+                action_data.update({"statusDetails": {"ignoreCount": ignore_count}})
```

Comment:
Just curious - what's an `ignore_count`?

Reply:
We let users choose to ignore indefinitely, or ignore until the event happens again 1, 10, 100, 1000, 10000 times. `ignore_count` is -1 for indefinitely.